x-pack/plugins/security_solution/cypress/tasks/create_new_rule.ts

```diff
@@ -273,22 +303,6 @@ export const selectThresholdRuleType = () => {
   cy.get(THRESHOLD_TYPE).click({ force: true });
 };
 
-export const waitForAlertsToPopulate = async () => {
-  cy.waitUntil(
-    () => {
-      refreshPage();
-      return cy
-        .get(SERVER_SIDE_EVENT_COUNT)
-        .invoke('text')
-        .then((countText) => {
-          const alertCount = parseInt(countText, 10) || 0;
-          return alertCount > 0;
-        });
-    },
-    { interval: 500, timeout: 12000 }
```

Comment:
Was there a reason for this to be here previously, i.e. timing reasons?  Are other tests relying on this?

Reply:
The method is on the same file but displayed in a different order. I just sorted the methods alphabetically.

Reply:
I didn't see parameters for interval and timeout in the other method.  Were the values here used at all?